<div align="center">
<img src="https://user-images.githubusercontent.com/32806475/191777062-b93ac448-44a7-46ff-a580-8e5cd4415574.png" width="320px">
</div>

# Capsid Community
近年、環境ウイルス分野のツールが数多く報告され、ツール選択は非常に困難です。
このように類似ツールが群雄割拠している現状は、新規参入の障壁や研究間比較の難化など、分野の大きな問題になっていると考えられます。  
Capsid Communityは、このような環境ウイルス分野のツールの情報をまとめ、(今後)比較することで、そのような課題を解決する場になることを目指したコミュニティです。
本コミュニティはまだまだ未熟ですが、ぜひ気軽にContributeしていただき、コミュニティ参加者の力によってより使いやすいものにしていければと思います。
Contributeの仕方がわからない場合は、Issueにコメントいただけますと、リストに加えます。カプシドタンパクが自己集合するように、研究に役立つ知識が自己集合するコミュニティとなることを願っています。  
(本コミュニティは微生物生態学会の環境ウイルス研究部会( http://www.microbial-ecology.jp/?page_id=7424 )の皆様によって支えられています)  
In recent years, a large number of bioinformatics tools have been reported in the field of environmental virology. Thus, the selection of tools for researchers is getting more and more difficult, and it is a high barrier to entering this field for newcomers. 
The “Capsid Community” aims to be a place where such issues can be resolved by gathering information of the tools and (in the future comparing them). Please feel free to contribute to Capsid Community. If you do not know how to contribute github, please comment on the issue and we will add you to the list. We hope that this community will be a place where knowledge useful for research can self-assemble, just as self-assemby of capsid proteins.  
(This community is supported by members of the Environmental Virus Research Group of the Japanese Society of Microbial Ecology)

# Tools
<!-- The tables below were automatically generated at the 2022-12-09 11:02:22.286526+09:00 -->

## ウイルス（ファージ）配列の予測(Prediction of viral and prophage sequences)
|Name|Prophage detection|Link|Reference link|Kazumax|Release|Note|
|:---|:---|:---|:---|:---|:---|:---|
|VirSorter2|○|https://github.com/jiarong/VirSorter2|https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00990-y|https://kazumaxneo.hatenablog.com/entry/2021/03/07/073000|2021|
|DeepMicrobeFinder|☓|https://github.com/chengsly/DeepMicrobeFinder|https://www.biorxiv.org/content/10.1101/2021.10.26.466018v1|https://kazumaxneo.hatenablog.com/entry/2021/11/12/080714|2021|
|VIBRANT|○|https://github.com/AnantharamanLab/VIBRANT|https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00867-0|https://kazumaxneo.hatenablog.com/entry/2020/06/14/175750|2021|AMG prediction|
|VirSorter|○|https://github.com/simroux/VirSorter|https://peerj.com/articles/985/|https://kazumaxneo.hatenablog.com/entry/2019/01/10/103631|2015|
|PPR-Meta||https://github.com/zhenchengfang/PPR-Meta|https://academic.oup.com/gigascience/article/8/6/giz066/5521157|https://kazumaxneo.hatenablog.com/entry/2019/10/09/234533|2019|
|Prophinder|○| http://aclame.ulb.ac.be/prophinder|https://academic.oup.com/bioinformatics/article/24/6/863/194494||2008|
|Phage_Finder||http://phage-finder.sourceforge.net/|https://academic.oup.com/nar/article/34/20/5839/3100473||2006|
|PhiSpy|○|https://github.com/linsalrob/PhiSpy|https://academic.oup.com/nar/article/40/16/e126/1027055||2012|
|PHASTER|○|https://phaster.ca/|https://academic.oup.com/nar/article/44/W1/W16/2499364||2016|PHAST, PHASTER and PHASTESTみたいな論文はあるが実際にはPHASTESTは全くリリースされていない(開発中？)|
|ProphET|○|https://github.com/jaumlrc/ProphET|https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0223364|
|MARVEL||https://github.com/LaboratorioBioinformatica/MARVEL|https://www.frontiersin.org/articles/10.3389/fgene.2018.00304/full||2018|
|VIROME||http://virome.dbi.udel.edu/|
|Metavir 2||http://metavir-meb.univ-bpclermont.fr/|https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-76|||expired?|
|VirFinder||https://github.com/jessieren/VirFinder|https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-017-0283-5|
|VirMiner||https://github.com/TingtZHENG/VirMiner|https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0657-y|https://kazumaxneo.hatenablog.com/entry/2019/06/03/221724|
|RabbitV||https://github.com/RabbitBio/RabbitUniq|https://academic.oup.com/bioinformatics/article/38/10/2932/6554196|https://kazumaxneo.hatenablog.com/entry/2022/04/30/004538|2022|
|viralFlye||https://github.com/Dmitry-Antipov/viralFlye|https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02566-x|https://kazumaxneo.hatenablog.com/entry/2022/02/24/234605|2022|ロングリード|
|fastv||https://github.com/OpenGene/fastv|https://academic.oup.com/bib/article/22/2/924/5917007]|https://kazumaxneo.hatenablog.com/entry/2021/07/10/224445|2021|Not for phages?|
|CheckV||https://bitbucket.org/berkeleylab/CheckV|https://www.nature.com/articles/s41587-020-00774-7|https://kazumaxneo.hatenablog.com/entry/2020/05/09/073000|2021|quality check|
|metaviralSPAdes||https://github.com/ablab/spades/tree/metaviral_publication||https://kazumaxneo.hatenablog.com/entry/2020/05/23/235524||viralVerify|
|ViralRecall||https://github.com/faylward/viralrecall|https://www.mdpi.com/1999-4915/13/2/150||2021|NCLDV|
|ViBE||https://github.com/DMnBI/ViBE|https://academic.oup.com/bib/article-lookup/doi/10.1093/bib/bbac204||2022|eukaryotic|
|hafeZ|○|https://github.com/Chrisjrt/hafeZ.|https://www.biorxiv.org/content/10.1101/2021.07.21.453177v1||2021|
|Phigaro|○|https://github.com/bobeobibo/phigaro|https://academic.oup.com/bioinformatics/article/36/12/3882/5822875||2020|
|PhageBoost|○|https://github.com/ku-cbd/PhageBoost|https://academic.oup.com/nargab/article/3/1/lqaa109/6066536#222619888||2020|
|DBSCAN-SWA|○|https://github.com/HIT-ImmunologyLab/DBSCAN-SWA/|https://www.frontiersin.org/articles/10.3389/fgene.2022.885048/full|https://kazumaxneo.hatenablog.com/entry/2020/08/20/230110|2022|
|DeePhage||https://github.com/shufangwu/DeePhage|https://academic.oup.com/gigascience/article/10/9/giab056/6366926||2021|
|ViromeScan||http://sourceforge.net/projects/viromescan/|https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2446-3||2016|
|DeepVirFinder||https://github.com/jessieren/DeepVirFinder|https://link.springer.com/article/10.1007/s40484-019-0187-4||2020|
|vRhyme||https://github.com/AnantharamanLab/vRhyme|https://academic.oup.com/nar/article/50/14/e83/6584432|||binning|
|Prophage Hunter||https://pro-hunter.bgi.com/|https://academic.oup.com/nar/article/47/W1/W74/5494712||2019|
|MetaPhage||https://github.com/MattiaPandolfoVR/MetaPhage|https://journals.asm.org/doi/10.1128/msystems.00741-22||2022|
|Deep6||https://github.com/janfelix/Deep6|https://www.biorxiv.org/content/10.1101/2022.09.13.507819v1||2022|Metatranscriptomic Sequences|
|Phanta||https://github.com/bhattlab/phanta|https://www.biorxiv.org/content/10.1101/2022.08.05.502982v1||2022|
|ViromeQC||https://github.com/SegataLab/viromeqc|https://www.nature.com/articles/s41587-019-0334-5||2022|Detecting contamination in viromes|
|PropagAtE|○|https://github.com/AnantharamanLab/PropagAtE|https://journals.asm.org/doi/10.1128/msystems.00084-22||2022|Detecting active prophages|
|PhageAI||https://github.com/AnantharamanLab/PropagAtE|https://www.biorxiv.org/content/10.1101/2020.07.11.198606v1||2022|
|BACPHLIP||https://github.com/adamhockenberry/bacphlip|https://peerj.com/articles/11396/||2021|
|SourceFinder||https://cge.food.dtu.dk/services/SourceFinder/|https://journals.asm.org/doi/10.1128/spectrum.02641-22||2022|Identification of Chromosomal, Plasmid, and Bacteriophage Sequences|
|genomad||https://github.com/apcamargo/genomad|||2022|Used in IMG/VR ver4|


## ゲノム(遺伝子)アノテーション(Genome/gene annotation tools)
|Name|Type|Link|Reference link|Kazumax|Release|Note|
|:---|:---|:---|:---|:---|:---|:---|
|DRAM|AMG setection|https://github.com/WrightonLabCSU/DRAM|https://academic.oup.com/nar/article/48/16/8883/5884738|https://kazumaxneo.hatenablog.com/entry/2022/02/10/000615|2020|
|VIBRANT|AMG prediction|https://github.com/AnantharamanLab/VIBRANT|https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00867-0|https://kazumaxneo.hatenablog.com/entry/2020/06/14/175750|2021|
|MetaPop|population analysis|https://github.com/metaGmetapop/metapop|https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01231-0|https://kazumaxneo.hatenablog.com/entry/2021/03/04/084538|2022|
|VIGOR|gene anotation|https://github.com/JCVenterInstitute/VIGOR4|https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-451|https://kazumaxneo.hatenablog.com/entry/2020/11/13/225137|2022(latest)|Eukaryotic virus?|
|VADR|gene anotation|https://github.com/nawrockie/vadr|https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3537-3||2020|
|pharokka|gene anotation|https://github.com/gbouras13/pharokka||https://kazumaxneo.hatenablog.com/entry/2022/07/03/220509||PHANOTATE×PHROGs|
|PHANOTATE|gene prediction|https://github.com/deprekate/PHANOTATE|https://academic.oup.com/bioinformatics/article/35/22/4537/5480131|https://kazumaxneo.hatenablog.com/entry/2019/05/30/073000|
|MetaPhage|gene anotation|https://github.com/MattiaPandolfoVR/MetaPhage|https://www.biorxiv.org/content/10.1101/2022.04.17.488583v1|https://kazumaxneo.hatenablog.com/entry/2022/04/23/124539|
|VirionFinder|virion proteins identification|https://github.com/zhenchengfang/VirionFinder|https://www.frontiersin.org/articles/10.3389/fmicb.2021.615711/full||2021|


## 配列のClassification (系統分類) Taxonomic classification of viral sequences
|Name|Link|Reference link|Kazumax|Release|Note|
|:---|:---|:---|:---|:---|:---|
|Vcontact2|https://bitbucket.org/MAVERICLab/vcontact2/wiki/Home|https://www.nature.com/articles/s41587-019-0100-8|https://kazumaxneo.hatenablog.com/entry/2019/09/21/235859|2019|
|ViPTree|https://www.genome.jp/viptree/|https://academic.oup.com/bioinformatics/article/33/15/2379/3096426||2017|
|VICTOR|https://ggdc.dsmz.de/victor.php||https://academic.oup.com/bioinformatics/article/33/21/3396/3933260|2017|
|VIRIDIC|http://rhea.icbm.uni-oldenburg.de/VIRIDIC/|https://www.mdpi.com/1999-4915/12/11/1268||2020|
|VirClust|http://rhea.icbm.uni-oldenburg.de/virclust/|https://www.biorxiv.org/content/10.1101/2021.06.14.448304v1||2021|
|GRAViTy|http://gravity.cvr.gla.ac.uk/|https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0422-7||2018|
|viruses_classifier|https://github.com/wojciech-galan/viruses_classifier|https://www.nature.com/articles/s41598-019-39847-2|https://kazumaxneo.hatenablog.com/entry/2019/03/15/073000|2019|
|VPF-Class|https://github.com/biocom-uib/vpf-tools|https://academic.oup.com/bioinformatics/article/37/13/1805/6104829||2021|
|Classiphage|http://appmibio.uni-goettingen.de/index.php?sec=sw|https://www.mdpi.com/1999-4915/11/2/195|
|VIRify|https://github.com/EBI-Metagenomics/emg-viral-pipelinee|
|ClassiPhages 2.0|http://appmibio.uni-goettingen.de/index.php?sec=sw|https://www.biorxiv.org/content/10.1101/558171v1|
|PhageClouds|https://phageclouds.ku.dk/|https://www.liebertpub.com/doi/full/10.1089/phage.2021.0008|


## 宿主予測 Host prediction
|Name|Link|Reference link|Kazumax|Release|Note|
|:---|:---|:---|:---|:---|:---|
|iPHoP|https://bitbucket.org/srouxjgi/iphop/src/main/|https://www.biorxiv.org/content/10.1101/2022.07.28.501908v1||2022|
|WIsH|https://github.com/soedinglab/wish|https://academic.oup.com/bioinformatics/article/33/19/3113/3964377||2017|
|SpacePHARER|https://github.com/soedinglab/spacepharer|https://academic.oup.com/bioinformatics/article/37/19/3364/6207963||2021|
|VirHostMatcher-Net|https://github.com/WeiliWw/VirHostMatcher-Net|https://academic.oup.com/nargab/article/2/2/lqaa044/5861484||2020|
|CHERRY|https://github.com/KennthShang/CHERRY|https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbac182/6589865||2022|
|RaFAH|https://sourceforge.net/projects/rafah/|https://www.biorxiv.org/content/10.1101/2020.09.25.313155v1|
|HostPhinder|https://cge.cbs.dtu.dk/services/HostPhinder/|https://www.mdpi.com/1999-4915/8/5/116|https://kazumaxneo.hatenablog.com/entry/2018/08/31/164441|
|PHISDetector|https://github.com/HIT-ImmunologyLab/PHISDetector|https://www.sciencedirect.com/science/article/pii/S1672022922000171||2022|
|vHULK|https://github.com/LaboratorioBioinformatica/vHULK|https://www.biorxiv.org/content/10.1101/2020.12.06.413476v1|
|ILMF-VH|https://github.com/liudan111/ILMF-VH|https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3082-0|
|PHISDetector|http://www.microbiome-bigdata.com/PHISDetector/|https://www.biorxiv.org/content/10.1101/661074v2|
|Host Taxon Predictor (HTP)|https://github.com/wojciech-galan/viruses_classifier|https://www.nature.com/articles/s41598-019-39847-2|
|ContigNet|https://github.com/tianqitang1/ContigNet|https://academic.oup.com/bioinformatics/article/38/Supplement_1/i45/6617519|


## CRISPR同定 Identification of host CRISPR sequces
|Name|Link|Reference link|Kazumax|Release|Note|
|:---|:---|:---|:---|:---|:---|
|CRISPRDetect|http://crispr.otago.ac.nz/CRISPRDetect/predict_crispr_array.html|https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2627-0|https://kazumaxneo.hatenablog.com/entry/2019/12/15/130706|2016|
|MinCED|https://github.com/ctSkennerton/minced|
|CRISPR Recognition Tool(CRT)|http://www.room220.com/crt/|https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-209||2007|
|crass|https://github.com/ctskennerton/Crass|
|CRISPRCasFinder|https://crisprcas.i2bc.paris-saclay.fr/CrisprCasFinder/Index|
|CRISPRCasTyper|https://crisprcastyper.crispr.dk/|https://www.liebertpub.com/doi/10.1089/crispr.2020.0059|
|AcrFinder|https://bcb.unl.edu/AcrFinder/|https://academic.oup.com/nar/article/48/W1/W358/5836766||2020|
|SpacePHARER|https://github.com/soedinglab/spacepharer|https://academic.oup.com/bioinformatics/article/37/19/3364/6207963||2021|
|CRT|http://www.room220.com/crt//|https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-209||2007|
|PILER-CR|http://www.drive5.com/pilercr.|https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-18||2007|
|CRISPRleader|https://github.com/BackofenLab/CRISPRleader|https://doi.org/10.1093/bioinformatics/btw454||2016|


## データベース Databases related to virology
|Name|Type|Link|Reference link|Release|Note|
|:---|:---|:---|:---|:---|:---|
|PHROG|Orthologue db|https://phrogs.lmge.uca.fr/|https://academic.oup.com/nargab/article/3/3/lqab067/6342220|2021|MMseqs2/hhblits search|
|pVOGs|Orthologue db|http://dmk-brain.ecn.uiowa.edu/pVOGs/downloads.html|https://academic.oup.com/nar/article/45/D1/D491/2333930|2016|Link expired?|
|Virus-Host DB|Host / Genomes|https://www.genome.jp/virushostdb/|https://www.mdpi.com/1999-4915/8/3/66|2022|
|ICTV|Taxonomy|https://ictv.global/taxonomy|https://academic.oup.com/nar/article/46/D1/D708/4508876|2021(initial 2018)|
|IMG/VR|Genomes|https://img.jgi.doe.gov/vr|https://academic.oup.com/nar/article/49/D1/D764/5952208|2021|
|IVirus|pipeline/tools|https://www.ivirus.us/|https://www.nature.com/articles/s43705-021-00083-3|2021|ver2|
|VirHostNet|virus/host molecular interactions|https://virhostnet.prabi.fr/|https://academic.oup.com/nar/article/43/D1/D583/2437900|2022|ver3|
|CRISPRCasdb|CRISPR cas|https://crisprcas.i2bc.paris-saclay.fr/|https://academic.oup.com/nar/article/48/D1/D535/5590637|2019|
|CrisprOpenDB|CRISPR cas|https://github.com/edzuf/CrisprOpenDB|https://academic.oup.com/nar/article/49/6/3127/6157093#233788514|2021|
|inphared|Genomes|https://github.com/RyanCook94/inphared)|https://www.liebertpub.com/doi/10.1089/phage.2021.0007|2021|
|VOG db|Orthologue db|https://vogdb.org/|
|VPFs|Orthologue db|https://portal.nersc.gov/dna/microbial/prokpubs/EarthVirome_DP/|https://www.nature.com/articles/nprot.2017.063|
|efam|Orthologue db|https://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/Zayed_efam_2020.1|https://academic.oup.com/bioinformatics/article/37/22/4202/6300514#312931372|
|ViPhOGs|Orthologue db|https://osf.io/2zd9r/|https://www.mdpi.com/1999-4915/13/6/1164|


## プラスミド様配列の検出 Plasmid detection
|Name|Link|Reference link|Kazumax(for Japanese beginner)|Release|Note|
|:---|:---|:---|:---|:---|:---|
|RFplasmid|https://github.com/aldertzomer/RFPlasmid|https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000683|https://kazumaxneo.hatenablog.com/entry/2020/09/29/233000|2021|
|PlasFlow|https://github.com/smaegol/PlasFlow|https://academic.oup.com/nar/article/46/6/e35/4807335|https://kazumaxneo.hatenablog.com/entry/2019/05/11/073000|2018|
|PlassClass|https://github.com/Shamir-Lab/PlasClass|https://academic.oup.com/nar/article/46/6/e35/4807335|https://kazumaxneo.hatenablog.com/entry/2019/10/21/073000|2020|
|metaplasmidSPAdes |https://github.com/ablab/spades/releases|https://genome.cshlp.org/content/29/6/961.long|https://kazumaxneo.hatenablog.com/entry/2019/06/05/073000|2019|
|PlasmidSeeker|https://github.com/bioinfo-ut/PlasmidSeeker|https://peerj.com/articles/4588/|https://kazumaxneo.hatenablog.com/entry/2018/05/28/203726|2018|
|Recycler|https://github.com/Shamir-Lab/Recycler|https://academic.oup.com/bioinformatics/article/33/4/475/2623362|https://kazumaxneo.hatenablog.com/entry/2017/08/27/233654|2018|

